tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -3179,10 +3183,342 @@ def test_forward_isfinite():
     _verify_infiniteness_ops(tf.is_finite, "isfinite")
 
 
+def _test_spop_placeholder_one():
+    with tf.Graph().as_default():
```

Comment:
Advice to use appropriate name instead of _one / _two ...etc.

Reply:
Will do. Thanks.

Reply:
Changed the test cases names appropriately.